tests/data_context/conftest.py

```diff
@@ -633,3 +634,23 @@ def datasource_config() -> DatasourceConfig:
             }
         },
     )
+
+
+@pytest.fixture
+def bearer_test_token() -> str:
+    return "6bb5b6f5c7794892a4ca168c65c2603e"
+
+
+@pytest.fixture
+def shared_called_with_request_kwargs(bearer_test_token) -> dict:
```

Comment:
This is pretty verbose.
Would this make sense as just ... `called_with_request_kwargs`?